airbyte-integrations/connectors/source-surveycto/integration_tests/abnormal_state.json

```diff
@@ -0,0 +1,5 @@
+{
+  "todo-stream-name": {
+    "todo-field-name": "todo-abnormal-value"
+  }
```

Comment:
set to a valid stream and field name with wrong state value to ensure the test runs correctly